crates/weaver_checker/README.md

```diff
@@ -105,10 +105,8 @@ deny[schema_evolution_violation("attr_removed", old_group.id, old_attr.id)] {
 }
 ```
 
-> [!NOTE]
```

Comment:
These [!NOTE] are valid GitHub Markdown Extensions, see the following blog post.

https://docs.github.com/fr/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts